src/components/study-pane.js

```diff
@@ -411,6 +454,22 @@ const StudyPane = (props) => {
                         values[0].threeWindingsTransformers
                     );
                     network.updateGenerators(values[0].generators);
+                    network.updateLoads(values[0].loads);
```

Comment:
Could be :
updatedEquipments.then((values) => {
                    network.updateSubstations(values.substations);
                    ...

Reply:
No, because a further US wil concern the lazy loading of a study ...

Reply:
ok I see.